tests/end_to_end_tests/journey_1/personal_details/section.py

```diff
@@ -373,6 +373,17 @@ async def run_checks_on_section(page: Page, asserts: AssertHelpers, helpers: Pag
     await asserts.number_of_errors(1)
     await asserts.error(field='statutory_declaration_date_year', message='Enter a date within the last 100 years')
 
+    # Enter a valid date that is before transition date

```

Comment:
I think it would be a little easier to read if we used the Transition Month and Year constants as input to the Statutory Declaration date. Then use (Transition Day -1) as the day field. This would show that the Statutory declaration date is 1 day prior to the dates used in the Transition date field.

Reply:
So transition date doesn't have a day so I've changed the month and year to show it's using the transition date minus one month which is invalid. Is that clearer?